.github/workflows/tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-24.04, windows-2022]
-        toolchain: [nightly, beta, stable, "1.63"]
+        toolchain: [nightly, beta, stable, "1.85"]
```

Comment:
Ideally, we should extract MSRV from the `rust-version` field instead of manually providing it.